recipes/elpigraph-python/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "elpigraph-python" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 15ac2dcefe4d2a0b475359d10471f0ec2e8c069100dc81208c924efe845231dd
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy >=1.16.2
+    - pandas >=0.23.4
+    - numba >=0.49.1 
+    - scikit-learn >=0.21.3
+    - scipy >=1.2.0
+    - python-igraph >=0.7.1
+    - matplotlib-base
+
+
+test:
+  imports:
+    - elpigraph
+    - elpigraph.src
```

Can we just add ``pip check`` here please?
```suggestion
    - elpigraph.src
  requires:
    - pip
  commands:
    - pip check
```